src/main/resources/config/case-data-fields.json

```diff
@@ -436,7 +436,8 @@
     "ExGratiaThirdRefundReason",
     "ExGratiaRefundAmount",
     "ExGratiaPaymentMethod",
-    "ExGratiaPaymentReferenceNumber"
+    "ExGratiaPaymentReferenceNumber",
+    "ExGratiaAdditionalInformation"
```

Comment:
Had a chat with Jospeh and this needs to be added to audit to show them in case_data extract.